atomium/templates/link/link.component.inc

```diff
@@ -27,7 +27,7 @@ function atomium_atomium_theme_link(array $existing, $type, $theme, $path) {
  * @SuppressWarnings(PHPMD.CyclomaticComplexity)
  */
 function atomium_preprocess_link(array &$variables, $hook) {
-  global $language_url;
+  $language_url = $GLOBALS['language_url'];
```

Comment:
If the variable is used only once, maybe it's not necessary to create a variable in the local scope.

Reply:
It ll be changed!